.github/workflows/major-version-updater.yml

```diff
@@ -0,0 +1,29 @@
+name: Major Version Updater
+# Whenever a new release is made, push a major version tag
+on:
+  release:
+    types: [ published ]
+
+jobs:
+  update-major-version-tag:
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout Repo
+        uses: actions/checkout@v3
+
+      - name: version
+        id: version
+        run: |
+          tag=${GITHUB_REF/refs\/tags\//}
+          version=${tag#v}
+          major=${version%%.*}
+          {
+            echo tag=${tag}
+            echo "version=${version}
+            echo "major=${major}" 
+          } >> "$GITHUB_OUTPUT"
```

Comment:
@zkoppert The linter forced this syntax, but I personally think this is a case of 'linter makes things less-common and more annoying', vs:

```bash
echo "tag=${tag}" >> "$GITHUB_OUTPUT"
echo "version=${version}" >> "$GITHUB_OUTPUT"
echo "major=${major}" >> "$GITHUB_OUTPUT"
```

Happy to keep it as is but thought I'd call it out.

Reply:
I agree with you here, I'm disabling that check and reverting to original syntax...

Reply:
Nice! Appreciate the change 👍

Reply:
Looks like I might not have the rc file in the correct place but i'll get there. 😅